draft-ietf-tls-dtls13.md

```diff
@@ -1128,11 +1128,15 @@ the length field of the original message.  An unfragmented message is
 a degenerate case with fragment_offset=0 and fragment_length=length.
 Each range MUST be delivered in a single UDP datagram.
 
-When a DTLS implementation receives a handshake message fragment, it
-MUST buffer it until it has the entire handshake message.  DTLS
+When a DTLS implementation receives a handshake message fragment corresponding
+to the next expected handshake message sequence number, it
+MUST buffer it until it has the entire handshake message. DTLS
 implementations MUST be able to handle overlapping fragment ranges.
 This allows senders to retransmit handshake messages with smaller
-fragment sizes if the PMTU estimate changes.
+fragment sizes if the PMTU estimate changes. Senders MUST NOT change 
+handshake message bytes upon retransmission. Receivers MAY check
```

Comment:
What handshake message bytes are you referring to here?

Reply:
This is just trying to reiterate that retransmission is literally just retransmitting the content, with no changes.  I don't think there are any specific bytes that one might be especially tempted to change.